keras_cv/utils/preset_utils.py

```diff
@@ -138,12 +153,23 @@ def check_preset_class(
 ):
     """Validate a preset is being loaded on the correct class."""
     config_path = get_file(preset, config_file)
-    with open(config_path) as config_file:
-        config = json.load(config_file)
+    try:
+        with open(config_path) as config_file:
+            config = json.load(config_file)
+    except:
+        raise ValueError(
+            f"The specified preset  `{preset}` is unknown. "
+            "Please check documentation to ensure the correct preset "
+            "handle is being used."
+        )
     cls = keras.saving.get_registered_object(config["registered_name"])
+    # Subclass checking
     if not isinstance(classes, (tuple, list)):
         classes = (classes,)
-    if cls not in classes:
+    # Alias subclass checking
+    if cls not in classes and not any(
+        issubclass(alias, cls) for alias in classes
```

Comment:
Actually shouldn't we reverse this? `issubclass(cls, Backbone)` (for example)?

Reply:
I've added both subclass and alias checking here, should be good now!